source/beamline/SGM/SGMEnergyCoordinator.cpp

```diff
@@ -556,7 +559,8 @@ void SGMEnergyCoordinator::onEnergySetpointPVChanged(double value)
 	if(energySetpointInitialized_)  {
 
 		qDebug() << QString("Forwarding energy setpoint value of %1 to energy control").arg(value);
-		energyControlCoordinator_->move(value);
+		AMControl* controlVersion = energyControlCoordinator_;
```

Comment:
It is to stop the initial setting of the energy during start up from being actually prompting the energy control from making a full move.
I did this because there may be a case where someone moves to a given energy with undulator tracking off, then for some reason restarts the coordinator. If this check wasn't here when the energy is calculated from the grating on start up, and this value written to the energy setpoint (to ensure it's at the right value to being with) the undulator (which because tracking was off during the initial move to this position isn't at the right spot) would begin to move. 

In short I didn't want a situation to arise where the undulator would begin to move just because the application restarted.